RotaryWheelControl/RotaryWheel.xaml.cs

```diff
@@ -143,6 +145,52 @@ public RotaryWheel()
             };
         }
 
+        private void SpinTo(int itemIndex, int durationInSec = -1)
+        {
+            Random r = new Random(DateTime.Now.Millisecond);
+            var angleFromYAxis = 360 - Angle;
+            SelectedItem = _pieSlices
```

Comment:
It looks like you are getting the currently selected pie slice? Why not use the `SelectedItem` getter?